applications/sdp/mspi/src/hrt/hrt.c

```diff
@@ -10,6 +10,18 @@
 /* Hardware requirement, to get n shifts SHIFTCNTB register has to be set to n-1*/
 #define SHIFTCNTB_VALUE(shift_count) (shift_count - 1)
 
+#define SPI_INPUT_PIN_NUM 2
+#define CNT1_INIT_VALUE	  1
+#define MSB_MASK (0xff000000)
+
+#define INPUT_SHIFT_COUNT (BITS_IN_WORD - BITS_IN_BYTE)
+
+/*
+ * Macro for calculating TOP value of CNT1. It should be twice as TOP value of CNT0
```

Comment:
Twice the top value is different from this formula, this should be explained.

Reply:
It is twice the CNT0 value, the catch here is that the value written to the register should be decreased by 1 (so when it should be 32, 31 should be written to the register itself). I'll add a note.